spec/unit/provider/package/windows_spec.rb

```diff
@@ -233,9 +233,9 @@
       end
     end
 
-    it "returns @installer_type if it is set" do
-      provider.new_resource.installer_type(:downeaster)
-      expect(provider.installer_type).to eql(:downeaster)
+    it "returns the resource's installer_type if it is set" do
+      provider.new_resource.installer_type(:nsis)
+      expect(provider.installer_type).to eql(:nsis)
     end
```

Comment:
The resource has validation on the `installer_type` which probably means that behavior changed after this test was written and it was never updated.